### PR TITLE
New screenshots and docs for granular permissions and updated org mem…

### PR DESCRIPTION
**Pull request by MichaelSavoia**
…bers/invites

### CODE REVIEW

File: docs/organizations/members.md [hunk 1]
```diff
@@ -1,52 +1,65 @@
 # Organization Members
 
-An organization can have multiple members, with limits on members and pending invitations depending on the organization's tier level. The organization's overview screen displays a list of current members, and the `Members` screen provides more details on each member. If you're an Administrator, that screen also displays a list of pending and expired invitations.
+An organization can have multiple members, with limits on members and pending invitations depending on the organization's tier level. The organization's overview screen displays a list of current members, the 'Members' screen allows an administrator to perform quick actions such as removing a member or renewing a pending invite, and the `Edit Member` screen provides more details on each member. If you're an Administrator, that screen also displays a list of pending and expired invitations.
```
Comment (cephirothdy2j) [inline docs/organizations/members.md]: By the final sentence, it's unclear which screen you're talking about; and, only administrators can access the latter two screens.

File: docs/organizations/members.md [hunk 2]
```diff
@@ -1,52 +1,65 @@
...
 * **Viewer**: A member with the Viewer role can see the contents of an organization, including all applications and dashboards owned by that organization, but can not change or modify anything about those applications, dashboards, or the organization itself. This extends into any application as well - with the Viewer role, a user can see devices and workflows inside an org-owned application, but will not be able to modify them.
-* **Editor**: A member with the Editor role can do everything a member with the Viewer role can do, plus ...
+* **Collaborator**: A member with the Collaborator role can do everything that a member with the Viewer role can do, plus ...
     * Modify applications or dashboards owned by the organization
-    * Create new applications or dashboards in that organization
     * Create or modify sub-resources within the org-owned application (e.g. devices, workflows)
+* **Editor**: A member with the Editor role can do everything a member with the Collaborator role can do, as well as create new application or dashboards in that organization.
```
Comment (cephirothdy2j) [inline docs/organizations/members.md]: create new applications

File: docs/organizations/members.md [hunk 3]
```diff
@@ -1,52 +1,65 @@
...
 
-Administrators can also remove any current organization member – including other administrators – by clicking the `Remove` (-) icon to the right of the user's name. This does not delete the user's Losant account; it simply cuts off their access to your organization. Any user removed from an organization can always be re-invited at a later date.
+Administrators can also remove any current organization member – including other administrators – by clicking the `Remove Member` button in the footer to the right of the `Update Member` button and confirm this action.
```
Comment (cephirothdy2j) [inline docs/organizations/members.md]: This is kind of a run-on sentence.

File: docs/organizations/members.md [hunk 4]
```diff
@@ -1,52 +1,65 @@
...
+
 ![Organization Members](/images/organizations/organization-members.png "Organization Members")
 
-Administrators can change the role of any organization member (other than themselves) from the `Members` page. The user's current role is displayed in a drop-down box to the right of their user info. Simply select a new role for the user to complete the change. It is not necessary to confirm your change, and it is very easy to change the user's role back immediately after your initial change.
+Administrators can change the role of any organization member (other than themselves) from the `Edit Member` page. This can be accessed by clicking on the member's email on either the `Organization Overview` or the `Members` screen. The user's current role is displayed in a drop-down box below their user info. Below this, administrators may also set the members granular permissions. Here administrators may select an individual application/dashboard as well as a permission that only applies to the selected asset. Simply select a new role for the user and optionally any granular permissions and click the `Update Member` button in the footer to complete the changes.
```
Comment (cephirothdy2j) [inline docs/organizations/members.md]: member's granular permissions (add an apostrophe).

File: docs/organizations/members.md [hunk 5]
```diff
@@ -1,52 +1,65 @@
...
+![Organization Invite Details](/images/organizations/organization-invite-details.png "Organization Invite Details")
+
+To revoke a pending invitation, click `Revoke Invitation` to the right of the invited user's email and confirm this action. Administrators may also revoke any invitation from the `Members` screen by clicking the `Remove` icon in the furthest right table column. If you need to change any of the invitee's roles, you will either have to wait for the user to accept the invitation and then change their access, or you will have to revoke the current invitation and send the user a new one.
 
 ## Expired Invitations
 
-Any invitation that was not accepted after seven days appears in the `Expired Invitations` section of the `Members` page. Invitations in this state do not count against your new invitation limit. Invitees whose invitation has expired can no longer join your organization.
+Any invitation that was not accepted after seven days has expired. Invitations in this state do not count against your new invitation limit. Invitees whose invitation has expired can no longer join your organization.
```
Comment (cephirothdy2j) [inline docs/organizations/members.md]: I'd clarify this a bit; they can no longer join the organization *using this invitation*; they can always be sent a new invitation (or a renewal).

**Review by cephirothdy2j — COMMENTED** [review]
Inline comments, plus were the images crunched?

**Comment by MichaelSavoia** [comment]
Yes, all images were run through imageOptim.

**Pull request merged**